AC3.2Unit2FinalAssessment/AC3.2Unit2FinalAssessment/ViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ViewController.swift
+//  AC3.2Unit2FinalAssessment
+//
+//  Created by Sabrina Ip on 10/6/16.
+//  Copyright © 2016 Sabrina Ip. All rights reserved.
+//
+
+import UIKit
+
+var currentValue = 5
+
+class ViewController: UIViewController {
```

Comment:
Needs to conform to UITextFieldDelegate and set `textField.delegate = self` to access delegate methods.